apps/src/templates/sectionProgress/StudentProgressSummaryCell.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { PropTypes, Component } from 'react';
+import { levelByLesson } from '@cdo/apps/code-studio/progressRedux';
+import {
+  sectionDataPropType,
+  scriptDataPropType,
+  studentLevelProgressPropType
+} from './sectionProgressRedux';
+import ProgressBox from '../sectionProgress/ProgressBox';
+import { LevelStatus } from '@cdo/apps/util/sharedConstants';
+
+export default class StudentProgressSummaryCell extends Component {
+  static propTypes = {
+    section: sectionDataPropType.isRequired,
+    studentId: PropTypes.number.isRequired,
+    stageId: PropTypes.number.isRequired,
+    scriptData: scriptDataPropType.isRequired,
+    studentLevelProgress: studentLevelProgressPropType.isRequired,
+  };
+
+  studentLevelProgressInStage(studentId, stageId) {
+    const { scriptData, studentLevelProgress } = this.props;
```

Comment:
Couldn't you also get studentId and stageId from props, instead of passing them in?  Or vice-versa?

Reply:
Very true, will update in next PR.